dev-support/docker-images/jupyter/Dockerfile

```diff
@@ -92,12 +92,14 @@ RUN wget --quiet https://repo.continuum.io/miniconda/Miniconda3-py37_${MINICONDA
     rm -rf /home/$NB_USER/.cache/yarn
 
 # Install latest sumbarine python sdk and notebook
-RUN pip install notebook==6.1.3 && \
-    git clone https://github.com/apache/submarine && \
-    pip install submarine/submarine-sdk/pysubmarine && \
-    cp submarine/submarine-sdk/pysubmarine/example/submarine_experiment_sdk.ipynb $HOME && \
-    cp -r submarine/submarine-sdk/pysubmarine/example/{data,deepfm_example.ipynb,deepfm.json} $HOME && \
-    rm submarine -rf
+RUN git clone https://github.com/apache/submarine && \
+pip install submarine/submarine-sdk/pysubmarine && \
+conda install nodejs && \
+conda install -c conda-forge jupyterlab jupyterlab-git && \
+jupyter lab build && \
+cp submarine/submarine-sdk/pysubmarine/example/submarine_experiment_sdk.ipynb $HOME && \
+cp -r submarine/submarine-sdk/pysubmarine/example/{data,deepfm_example.ipynb,deepfm.json} $HOME && \
+rm submarine -rf
```

Comment:
```suggestion
    pip install submarine/submarine-sdk/pysubmarine && \
    conda install nodejs && \
    conda install -c conda-forge jupyterlab jupyterlab-git && \
    jupyter lab build && \
    cp submarine/submarine-sdk/pysubmarine/example/submarine_experiment_sdk.ipynb $HOME && \
    cp -r submarine/submarine-sdk/pysubmarine/example/{data,deepfm_example.ipynb,deepfm.json} $HOME && \
    rm submarine -rfpip install submarine/submarine-sdk/pysubmarine && \
    conda install nodejs && \
    conda install -c conda-forge jupyterlab jupyterlab-git && \
    jupyter lab build && \
    cp submarine/submarine-sdk/pysubmarine/example/submarine_experiment_sdk.ipynb $HOME && \
    cp -r submarine/submarine-sdk/pysubmarine/example/{data,deepfm_example.ipynb,deepfm.json} $HOME && \
    rm submarine -rf
```

Reply:
Btw, Why do we need to install `nodejs`? Are we gonna use it?

Reply:
According to the description of [Jupyterlab](https://github.com/jupyterlab/jupyterlab-git),  it needs NodeJS to build the extension package.